### ISSUE TITLE
s6-rc: fatal: unable to take locks: Permission denied

### ISSUE THREAD

**Issue by Mist-Hunter** [PrivateBin/docker-nginx-fpm-alpine#61]
The 'latest' tag is throwing the following error:
s6-rc: fatal: unable to take locks: Permission denied

Same container config rolled back to tag '1.3.5' works fine. 

Thanks for this great container / software. I've been using it for years and this is the first issue I've encountered. Hopefully not user error :P

This occurs with any UID I've tried so far (0, default and 1000)

**Comment by elrido** [comment]
This should be a duplicate of #59, but I can't reproduce this with the latest image version, neither with nor without read-only root. Could you provide some more details on what CPU architecture and docker version (or other runtime) you are using?

**Comment by Mist-Hunter** [comment]
Docker version 20.10.8, build 3967b7d

And I just remembered I'm running Bullseye! Though from the looks of #59 it might not be related.

Debian GNU/Linux 11 (bullseye) x86_64
Kernel: 5.10.0-8-amd64

**Comment by elrido** [comment]
That is odd, I'm running the following on Ubuntu 20.04 LTS:
```
$ docker info 2>&1 | grep Version
 Server Version: 20.10.2
 Cgroup Version: 1
 Kernel Version: 5.4.0-80-generic
$ docker pull privatebin/nginx-fpm-alpine:latest
latest: Pulling from privatebin/nginx-fpm-alpine
Digest: sha256:536a2989e2738971e8b250753af8124ff91ae7e65fbdb77b8fffef73ef1986b0
Status: Image is up to date for privatebin/nginx-fpm-alpine:latest
docker.io/privatebin/nginx-fpm-alpine:latest
$ docker run -ti --rm privatebin/nginx-fpm-alpine:latest
s6-rc: info: service php-fpm8: starting
s6-rc: info: service php-fpm8 successfully started
s6-rc: info: service nginx: starting
s6-rc: info: service nginx successfully started
[14-Aug-2021 19:01:05] NOTICE: fpm is running, pid 21
[14-Aug-2021 19:01:05] NOTICE: ready to handle connections
```
Did running the image with `--read-only` change the behaviour for you?

**Comment by Mist-Hunter** [comment]
Adding:
--read-only 

Fixed the issue for me.

**Comment by elrido** [comment]
Could you reproduce this problem pulling the latest image or could you provide the image ID so we can be sure that you aren't using an older "latest" image that had this bug, that got resolved in #59? I'm unable to reproduce this in the currently published images (latest or otherwise).

**Comment by Mist-Hunter** [comment]
I removed the the '--read-only' flag and it is now loading fine. Thanks!
```
[17-Aug-2021 12:20:18] NOTICE: ready to handle connections,
[17-Aug-2021 12:20:18] NOTICE: fpm is running, pid 21,
s6-rc: info: service nginx: starting,
s6-rc: info: service nginx successfully started,
s6-rc: info: service php-fpm8 successfully started,s6-rc: info: service php-fpm8: starting
```
